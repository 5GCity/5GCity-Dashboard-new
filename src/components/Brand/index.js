/**
 * Brand Component
 *
 * @author Hugo Fonseca <hfonseca@ubiwhere.com>
 */
import React from 'react'
import styled from 'styled-components'
import { Logo } from 'components/Icons'

export default ({ children, ...props }) => (
  <Wrapper {...props}><Logo /></Wrapper>
)

const Wrapper = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  width:100%;
  height: 88px;
  background-color: #FFFFFF;
  box-shadow: inset 0 -1px 0 0 rgba(137,151,159,0.15);
`
