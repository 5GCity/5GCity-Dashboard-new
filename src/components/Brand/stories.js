/**
 * Brand Component Stories
 * Please write a description or remove this line
 *
 * @author Your Name <youremail@ubiwhere.com>
 */
import React from 'react'
import { storiesOf } from '@storybook/react'

import Brand from './index'

const State1 = () => (
  <Brand />
)

storiesOf('Brand', module)
  .add('primary', State1)