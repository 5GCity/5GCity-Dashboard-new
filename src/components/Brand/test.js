/**
 * Brand Component Tests
 *
 * @author Hugo Fonseca <hfonseca@ubiwhere.com>
 */
import React from 'react'
import renderer from 'react-test-renderer'
import Brand from './index'
import 'jest-styled-components'

it('renders correctly', () => {
  const tree = renderer
    .create(<Brand />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
