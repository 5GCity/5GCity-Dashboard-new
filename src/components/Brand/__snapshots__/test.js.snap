// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  width: 100%;
  height: 88px;
  background-color: #FFFFFF;
  box-shadow: inset 0 -1px 0 0 rgba(137,151,159,0.15);
}

<div
  className="c0"
/>
`;
